package/gluon-mesh-olsrd/usr/lib/autoupdater/abort.d/10olsrd

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+. /lib/gluon/autoupdater/lib.sh
```

Comment:
These files under `package/gluon-mesh-olsrd/usr/` are unused, and there are similar files under `package/gluon-mesh-olsrd/files/`.